src/app/Dashboard/components/DashboardComponent.js

```diff
@@ -42,49 +46,80 @@ export class DashboardComponent extends Component {
     pointsEarned: PropTypes.number,
     activitiesLogged: PropTypes.number,
     fetchUserActivites: PropTypes.func,
+    loadCategories: PropTypes.func,
   };
 
   componentDidMount() {
-    const { fetchUserActivites } = this.props;
+    const { fetchUserActivites, loadCategories } = this.props;
     const token = getToken();
     const userInfo = getUserInfo(token);
     this.setState({ user: userInfo });
     fetchUserActivites(userInfo.id);
+    loadCategories();
+  }
+
+  openModalLoginPointsHandler = () => {
+    const { logPoints } = this.state;
+    this.setState({
+      logPoints: true,
+    });
+    console.log('logpoints', logPoints);
   }
 
   render() {
     const {
       error, society, loading, pointsEarned, activitiesLogged,
     } = this.props;
-    const { user } = this.state;
+    const { user, logPoints } = this.state;
+    console.log('before click', logPoints);
```

Comment:
Bump this.

Reply:
@kimani-ndegwa I have removed the console log and all the others.